### PR TITLE
[ML] Read scores from downloaded vocabulary for XLM Roberta tokenizers

### PR DESCRIPTION
**Pull request by davidkyle**
The model downloaded was not aware of the `scores` field in the vocabulary document which is required for the XLM Roberta tokenizer. This change now incorporates the `scores` field which is required for downloading and deploying XLM Roberta models

The max size of the vocab file download is bumped from 10MB to 20MB as the XLM Roberta tokenizer vocab is larger than 10MB and was being truncated.

### CODE REVIEW

File: x-pack/plugin/ml-package-loader/src/main/java/org/elasticsearch/xpack/ml/packageloader/action/TransportLoadTrainedModelPackage.java [hunk 1]
```diff
@@ -197,8 +197,8 @@ public CancellableTask createTask(long id, String type, String action, TaskId pa
         }, false);
     }
 
-    private static void recordError(Client client, String modelId, AtomicReference<Exception> exceptionRef, Exception e) {
-        logAndWriteNotificationAtError(client, modelId, e.toString());
+    private static void recordError(Client client, String modelId, AtomicReference<Exception> exceptionRef, ElasticsearchException e) {
+        logAndWriteNotificationAtError(client, modelId, e.getDetailedMessage());
```
Comment (davidkyle) [inline x-pack/plugin/ml-package-loader/src/main/java/org/elasticsearch/xpack/ml/packageloader/action/TransportLoadTrainedModelPackage.java:201]: logs the root cause if there is an nested exception.

File: x-pack/plugin/ml-package-loader/src/main/java/org/elasticsearch/xpack/ml/packageloader/action/ModelLoaderUtils.java [hunk 2]
```diff
@@ -114,32 +116,36 @@ static InputStream getInputStreamFromModelRepository(URI uri) throws IOException
...
-                    : Collections.emptyList();
-                List<String> merges = vocabAndMerges.containsKey(MERGES)
-                    ? vocabAndMerges.get(MERGES).stream().map(Object::toString).collect(Collectors.toList())
-                    : Collections.emptyList();
-
-                return Tuple.tuple(vocabulary, merges);
+    static VocabularyParts loadVocabulary(URI uri) {
+        if (uri.getPath().endsWith(".json")) {
+            try {
+                InputStream vocabInputStream = getInputStreamFromModelRepository(uri);
```
Comment (jonathan-buttner) [inline x-pack/plugin/ml-package-loader/src/main/java/org/elasticsearch/xpack/ml/packageloader/action/ModelLoaderUtils.java:122]: We weren't doing this prior to this PR but should we use a try-with-resources to wrap the inputstream and ensure it gets closed if an exception occurs?

**Review by jonathan-buttner — APPROVED** [review]

File: x-pack/plugin/ml-package-loader/src/main/java/org/elasticsearch/xpack/ml/packageloader/action/ModelLoaderUtils.java (hunk 2 above)
Comment (davidkyle) [reply x-pack/plugin/ml-package-loader/src/main/java/org/elasticsearch/xpack/ml/packageloader/action/ModelLoaderUtils.java]: Good point, I've made that change

**Comment by davidkyle** [comment]
@elasticmachine update branch

**Pull request merged**